ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/workers-rhel8/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,42 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8-provision
+  steps: 
+  - ref: openshift-cluster-bot-rbac
+  - ref: gcp-provision-vpc
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - chain: mirror-images-by-oc-adm
+  - ref: ipi-conf
+  - ref: ipi-conf-mirror
+  - ref: ipi-conf-gcp
+  - ref: ipi-install-install
```

Comment:
I think we can replace the line 4 ~ 12 with `chain: cucushift-installer-rehearse-gcp-ipi-disconnected-provision` here.